images/build/Dockerfiles/vso.slim.Dockerfile

```diff
@@ -0,0 +1,252 @@
+FROM githubrunners-buildpackdeps-focal AS main
```

Comment:
Is this slim image asked by the VSO team? I see this image removed some dotnet, python versions from the vso image, other than that I didn't see lots of difference from vso.Dockerfile image. Is this one necessary

Reply:
yes .. this was  asked by them ... they will let us know if they are going to migrate to this slimmer version .. for now this is necessary

Reply:
why do we base the VSO image on the GH runners one?